WEB_SERVICE/WEB/_HIGH/HOSTNAME_MISMATCH/description.md

```diff
@@ -0,0 +1,15 @@
+A hostname mismatch in an SSL/TLS certificate occurs when the domain name specified in the certificate does not match the actual hostname of the server. This mismatch can lead to security warnings in browsers and potential vulnerabilities to man-in-the-middle attacks.
+
+Key points about hostname mismatches:
```

Comment:
Key point style is not typical to our KB.

Reply:
Removed it.